### ISSUE TITLE
 A.I.M., Механоиды main menu not drawn

### ISSUE THREAD

**Issue by lukasva77** [crosire/d3d8to9#189]
I have discovered your wrapper due to gta vice city makes everything better more fps and stutter is gone but invisible bink video but it seems to be a known issue

so I've gone looking in my library for more dx8 games

Advanced Battlegrounds: The Future of Combat or Chrome: Gold Edition also increases fps and works fine I checked just the opening of the first level but whatever

but the game in the title fails in the main menu

just reporting if I find more games I will open more issues

**Comment by WinterSnowfall** [comment]
> but the game in the title fails in the main menu

I've checked the game and it seems to work fine, as long as you don't open the Options pop-up from the main menu. Unfortunately, that relies on the device being lost during window focus loss to work properly which may not behave properly on modern Windows, but as long as you set the options beforehand (using `SetOptions.exe`) and don't open the Options from the main menu, you shouldn't run into any issues.

**Comment by WinterSnowfall** [comment]
> Tested this a bit so the problem occurs from changing the graphics setting to anisotropic filtering default is trilinear
> 
> I didn't think that graphic options could do this but WinterSnowfall saying it works fine I had to check

Strange. I have it set to AF x16 and it's fine.

**Comment by WinterSnowfall** [comment]
> also a really useless menu item cause can just run SetOptions.exe but whatever

Agreed it's not a great design. But I think I know what might be causing the crash - at least I have some ideas to try.

As to the anisotropy setting, not sure what's going on there, as I can't reproduce the problem.
